actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -311,8 +311,9 @@ def button_to(name = nil, options = nil, html_options = nil, &block)
         form_options[:action] = url
         form_options[:'data-remote'] = true if remote
 
-        request_token_tag = if form_method == 'post'
-          token_tag(nil, form_options: form_options)
```

Comment:
The only change we need is in this line to `token_tag(nil, form_options: { action: url, method: method })`